cpp/ql/src/experimental/Security/CWE/CWE-758/DangerousUseMallocWithZeroSize.qhelp

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>The behavior of the malloc function is not defined when the size value is zero, so it is not possible to evaluate the correctness only by the result of the operation. Add an estimate for the lower bound of the size.</p>
+
+</overview>
+
+<example>
+<p>The following example shows the erroneous and fixed ways to use the malloc function.</p>
+<sample src="DangerousUseMallocWithZeroSize.cpp" />
+
+</example>
+<references>
+
+<li>
+  CERT Coding Standard:
+  <a href="https://wiki.sei.cmu.edu/confluence/display/c/MSC15-C.+Do+not+depend+on+undefined+behavior">MSC15-C. Do not depend on undefined behavior - SEI CERT C Coding Standard - Confluence</a>.
```

Comment:
Is there no reference that is more specific?
That reference is not adding much new information IMHO.

Reply:
Do you think this link is not correct?

Reply:
How about https://wiki.sei.cmu.edu/confluence/display/c/MEM04-C.+Beware+of+zero-length+allocations?

Reply:
@intrigus-lgtm  thanks